src/biguint.rs

```diff
@@ -2264,6 +2264,11 @@ pub(crate) fn biguint_from_vec(digits: Vec<BigDigit>) -> BigUint {
 }
 
 impl BigUint {
+    /// The value 0 as a `BigUint`
+    pub const ZERO: BigUint = BigUint {
```

Comment:
For MSRV, these new constants should be guarded by a `cfg`. You can use `autocfg::emit_constant_cfg` testing something like `Vec::<u32>::new()` in the build script. (Ditto on `BigInt`.)